bridge/src/commonMain/kotlin/ru/herobrine1st/matrix/bridge/telegram/TelegramWorker.kt

```diff
@@ -54,14 +54,32 @@ class TelegramWorker(
         actorId: TelegramActorId,
         id: ChatId
     ): RemoteRoom<ChatId> {
-        TODO("Not yet implemented")
+        val bot = getBot(actorId)
+        val chat = withContext(Dispatchers.IO) { bot.getChat(id).get() }
+
+        return RemoteRoom(
+            id = id,
+            isDirect = chat.type == "private",
+            displayName = chat.title ?: chat.username ?: "Unknown"
+        )
     }
 
     override fun getRoomMembers(
         actorId: TelegramActorId,
         remoteId: ChatId
-    ): Flow<Pair<UserId, RemoteUser<UserId>?>> {
-        TODO("Not yet implemented")
+    ): Flow<Pair<UserId, RemoteUser<UserId>?>> = flow {
+        val bot = getBot(actorId)
+
+        // Получаем всех адимнов чата (cringe, потом надо будет что-то придумать)
+        val members = withContext(Dispatchers.IO) {
+            bot.getChatAdministrators(remoteId).get()
+        }
+
+        for (member in members) {
+            val userId = UserId(member.user.id)
+            val remoteUser = RemoteUser(userId, member.user.username ?: "Unknown")
```

Comment:
```suggestion
            val remoteUser = RemoteUser(userId, member.user.firstName)
```
Использование firstName избавляет от необходимости обработки null-случая, поскольку является обязательным